distributions/demo-server/build.gradle

```diff
@@ -134,19 +134,37 @@ shadowJar {
 
 shadowJar.dependsOn(createSecretsFile, createApiFile)
 
-task createDemoServerDockerfile(type: Dockerfile) {
-    description = 'Builds the Demo Server Dockerfile'
+task createDemoDockerfile(type: Dockerfile) {
```

Comment:
nit: how about just createDockerfile?

before we were doing both client and server stuff, hence the different names